app/buyers/views/buyers.py

```diff
@@ -127,6 +127,10 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     sections = content.summary(brief)
     delete_requested = True if request.args.get('delete_requested') else False
 
+    content_loader.load_messages(brief['frameworkSlug'], ['urls'])
+    call_off_contract_url = content_loader.get_message(brief['frameworkSlug'], 'urls', 'call_off_contract_url')
```

Comment:
It will return `None` if it can't find the key it's after and the app stays up. You get a weird thing on the requirements page though. If for example the contract url wasn't set and so `call_off_contract_url` was set to None, the link would point at `http://localhost/buyers/frameworks/digital-outcomes-and-specialists/requirements/digital-specialists/None`.

I think this is ok? It just means that new frameworks need url's set?